modules/installation-osp-verifying-external-network.adoc

```diff
@@ -18,7 +18,18 @@ endif::[]
 The {product-title} installer requires external network access. You must provide an external network value to it, or deployment fails. Before you run the installer, verify that a network with the External router type exists in {rh-openstack-first}.
 
 .Prerequisites
-* https://docs.openstack.org/neutron/rocky/admin/config-dns-res.html#case-2-dhcp-agents-forward-dns-queries-from-instances[Configure OpenStack's networking service to have DHCP agents forward instances' DNS queries]
+
+* On {rh-openstack}, the `NeutronDhcpAgentDnsmasqDnsServers` parameter must be configured to allow DHCP agents to forward instances' DNS queries. One way to set this parameter is to:
+.. link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/advanced_overcloud_customization/sect-understanding_heat_templates#sect-Environment_Files[Create a new environment file] in the template directory.
+.. Provide link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/overcloud_parameters/index#networking-neutron-parameters[parameter values] in the file. For example:
++
+.`neutron-dhcp-agent-dnsmasq-dns-servers.yaml`
```

Comment:
Might need to add a hard return and + following ".`neutron-dhcp-agent-dnsmasq-dns-servers.yaml`". The formatting is wonky in the build.

Reply:
The code block looks great! For some reason, this one is building wonky still. I'm sorry. Might be the `` formatting. 

Maybe changing it to ".Sample `neutron-dhcp-agent-dnsmasq-dns-servers.yaml` file" or something similar?